Makefile

```diff
@@ -66,7 +66,7 @@ lint: $(VIRTUALENV_BIN)/flake8
 	$(VIRTUALENV_BIN)/flake8 gfdatasource/gfdatasource grafanalib
 
 test: $(VIRTUALENV_BIN)/py.test
-	$(VIRTUALENV_BIN)/py.test --junitxml=$(JUNIT_XML)
+	$(VIRTUALENV_BIN)/py.test --junitxml=$(JUNIT_XML) --cov=grafanalib --cov-report term-missing
```

Comment:
Nice! What do we need to do to get coverage stats as a README shield?

Reply:
I just used coverage as a way of figuring out if I got all the classes covered by tests (and thus by parse methods) ;-)
We can probably do some neat stuff. eg. export a html version.